src/pages/Survey/Survey.Api.ts

```
@@ -1,5 +1,6 @@
 import fetchingWithToast from "@/lib/customFetch";
 import type { SurveyFormValues } from "./Survey.schema";
+import { InterestField } from "@/types/api/survey";
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)